### PR TITLE
clienttoken to header, start mergedcal activity

### PR DESCRIPTION
**Pull request by Phaedra144**

### CODE REVIEW

File: app/src/main/java/com/greenfox/kalendaryo/LoginActivity.java [hunk 1]
```diff
@@ -107,12 +110,13 @@ public void onFailure(Call<KalUser> call, Throwable t) {
...
-    private void editSharedPref(String email, String userName, String token) {
+    private void editSharedPref(String email, String userName, String accessToken, String clientToken) {
         sharedPref = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
         editor = sharedPref.edit();
         editor.putString("email", email);
-        editor.putString("username", userName);
-        editor.putString("token", token);
+        editor.putString(email + "username", userName);
+        editor.putString(email + "accesstoken", accessToken);
+        editor.putString("clienttoken", clientToken);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/kalendaryo/LoginActivity.java]: refactor this using our own sharedpreferences

File: app/src/main/java/com/greenfox/kalendaryo/MainActivity.java [hunk 2]
```diff
@@ -43,32 +87,90 @@ public void onClick(View view) {
...
+                    adapter.add(kalendar);
+                }
+            }
+
+            @Override
+            public void onFailure(Call<KalendarsResponse> call, Throwable t) {
+                t.printStackTrace();
+            }
+        });
+    }
```
Comment (Phaedra144) [inline app/src/main/java/com/greenfox/kalendaryo/MainActivity.java:165]: Please remove whitespaces in this method.

File: app/src/main/java/com/greenfox/kalendaryo/models/KalAuth.java [hunk 3]
```diff
@@ -9,6 +9,8 @@
     private String authCode;
     private String email;
     private String displayName;
+    private String accessToken;
```
Comment (Phaedra144) [inline app/src/main/java/com/greenfox/kalendaryo/models/KalAuth.java]: accessToken is never used, please remove if it's not necessary.

File: app/src/main/java/com/greenfox/kalendaryo/adapter/CalendarAdapter.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import android.widget.TextView;
+
+import com.greenfox.kalendaryo.R;
+import com.greenfox.kalendaryo.models.Kalendar;
+
+/**
+ * Created by barba on 02/01/2018.
+ */
+
+public class CalendarAdapter extends ArrayAdapter<Kalendar> {
```
Comment (Phaedra144) [inline app/src/main/java/com/greenfox/kalendaryo/adapter/CalendarAdapter.java]: Can you please change the name to KalendarAdapter consequently?

**Pull request merged**